.evergreen/config.yml

```diff
@@ -361,6 +361,32 @@ functions:
            PYTHON_BINARY=${PYTHON_BINARY} sh ${PROJECT_DIRECTORY}/.evergreen/run-doctests.sh
 
   "run tests":
+    # If testing encryption start the mock KMS servers.
+    # First create the virtualenv and install dependencies.
+    - command: shell.exec
```

Comment:
Instead of doing this here, can't we do this in `run-tests.sh` gated under a check on `TEST_ENCRYPTION`? In config.yml under `"run tests"` we seem to limit ourselves to setting up environment variables which then get used by `run-tests.sh`. It'll be nice to stick to that...

Reply:
Done. That's way simpler.